CHANGELOG.md

```diff
@@ -7,6 +7,8 @@ Changelog continues [here](https://github.com/mobxjs/mobx/blob/master/CHANGELOG.
 * Fixed [#1534](Fixes https://github.com/mobxjs/mobx/issues/1534):  @computed({keepAlive: true}) no long calculates before being accessed.
 * Added the `$mobx` export symbol for MobX 5 forward compatibity
 
+* onBecome(Un)Observed didn't trigger when using number as key of observable map. Fixes [#2067](https://github.com/mobxjs/mobx/issues/2067).
```

Comment:
```suggestion
```
Don't worry about changelog here, this is for much older version, we maintain only `master` version.

Reply:
got it :D